proto/reaction.proto

```diff
@@ -318,11 +349,40 @@ message Vessel {
       CUSTOM = 1;
       NONE = 2;
       OVEN_DRIED = 3;
+      FLAME_DRIED = 4;
+      EVACUATED_BACKFILLED = 5;
+      PURGED = 6;
     }
   }
-  VesselPreparation.VesselPreparationType preparation = 5;
+  repeated VesselPreparation.VesselPreparationType preparations = 5;
   string preparation_details = 6;
-  Volume volume = 7;  // Size (volume) of the vessel.
+  message VesselAttachment {
```

Comment:
For long-ish messages like this, I suggest moving them to top-level and adding a message-level comment describing the proto.

Reply:
Done.